collector.go

```diff
@@ -39,9 +39,9 @@ func oidToList(oid string) []int {
 func ScrapeTarget(target string, config *config.Module) ([]gosnmp.SnmpPDU, error) {
 	// Set the options.
 	snmp := gosnmp.GoSNMP{}
-	snmp.Retries = 3
-	snmp.MaxRepetitions = 25
-	snmp.Timeout = time.Second * 60
+	snmp.Retries = config.Retries
+	snmp.MaxRepetitions = config.MaxRepititions
+	snmp.Timeout = time.Second * config.Timeout
```

Comment:
As I understand it, up to `snmp.Retries` are made within the time specified by `snmp.Timeout`. Should I understand this differently? It looks to me like it uses the `snmp.Timeout` to set a final timeout then the GoSNMP library divides the `snmp.Timeout` value by the `snmp.Retries` value for the amount of time to wait for a response to an individual attempt. Technically speaking the GoSNMP library may not fulfill all the specified attempts because the final timeout could occur prior to all of the requests being performed (granted it is unlikely). [See here.](https://github.com/soniah/gosnmp/blob/master/marshal.go#L105)

If my understanding is correct then what you are really saying is that you think that a Timeout specified in a config file should do what the user expects (which you seem to think is time allowed per attempt) and not follow the rules of the underlying code. If that is what you'd prefer it can be done that way. Let me know.